docs/source/release.rst

```diff
@@ -5,6 +5,54 @@
 Change Log
 ==========
 
+Release 2.7 - 4/14/2022
+-----------------------
+
+Documentation
+^^^^^^^^^^^^^
+A full-size upgrade of the documentation, with the following significant improvements in the reading experience, practical tutorials, and examples:
+
+* Reorganized the document structure with a new document template. (`Upgraded doc entry <https://nni.readthedocs.io/en/v2.7>`__)
+* Add more friendly tutorials with jupyter notebook.(`New Quick Starts <https://nni.readthedocs.io/en/v2.7/quickstart.html>`__)
+
+Hyper-Parameter Optimization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* [Improvement] TPE and random tuners will not generate duplicate hyperparameters anymore.
+* [Improvement] Most Python APIs now have type annotations.
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Jointly search for architecture and hyper-parameters: ValueChoice in evaluator (`doc <https://nni.readthedocs.io/en/v2.7/reference/nas/search_space.html#valuechoice>`__)
+* Support composition (transformation) of one or several value choices (`doc <https://nni.readthedocs.io/en/v2.7/reference/nas/search_space.html#valuechoice>`__)
+* Enhanced Cell API (``merge_op``, preprocessor, postprocessor) (`doc <https://nni.readthedocs.io/en/v2.7/reference/nas/search_space.html#cell>`__)
+* Repeat depth now allows a ValueChoice (`doc <https://nni.readthedocs.io/en/v2.7/reference/nas/search_space.html#repeat>`__)
```

Comment:
The argument `depth` in the `Repeat` API allows ValueChoice

Reply:
fixed